test/system/src/main/java/io/pravega/test/system/framework/kubernetes/K8sClient.java

```diff
@@ -498,11 +500,12 @@ public void deletePVC(String namespace, String name) {
         Retry.RetryAndThrowConditionally retryConfig = retryWithBackoff
                 .retryWhen(t -> {
                     Throwable ex = Exceptions.unwrap(t);
-                    if (ex.getCause() instanceof IOException) {
+                    if (ex.getCause() instanceof IOException && !(ex.getCause() instanceof UnknownHostException)) {
```

Comment:
It is odd that we are not handling these IO exceptions in any way. What reason do we have to believe that they are benign?

Reply:
In case of IO exception retry is performed a bounded number of times.